uint8 GRASPER_IDLE = 0
uint8 GRASPER_CLOSE = 1
uint8 GRASPER_OPEN = 2

Header  header
float32 base
float32 joint0
float32 joint1
float32 grasperRoll
float32 grasperPitch
int8 grasperEnd
